src/test/java/harness/TestConfig.java

```diff
@@ -0,0 +1,80 @@
+package harness;
+
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.utils.FileUtils;
+import bio.terra.cli.utils.JacksonMapper;
+import com.google.common.base.Preconditions;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Test config that can vary between Terra deployments. */
+public class TestConfig {
+  private static final Logger logger = LoggerFactory.getLogger(TestConfig.class);
+
+  // Some CLI tests directly create external resources, eg FineGrainedAccessGcsObjectReference.java
+  private String projectForExternalResources;
+
+  // Whether to use janitor to clean up external resources. For more about terra-cli and Janitor,
+  // see PF-886.
+  private boolean useJanitorForExternalResourcesCreatedByTests;
+
+  public static TestConfig get() {
```

Comment:
Perhaps you can create a singleton object here, so we don't load from disk & validate every time a tests calls `get()`.

Reply:
Done, thanks.